tests/Orckestra.Composer.Cart.Tests/Providers/EditingOrderProvider_IsOrderEditable.cs

```diff
@@ -109,5 +111,55 @@ protected Order CreateOrderWithShipments(List<Shipment> shipments)
                 }
             };
         }
+
+        [Test]
+        [TestCase("Canceled", false, false, null, false)]
+        [TestCase("InProgress", true, false, "New", true)]
+        [TestCase("InProgress", false, true, "New",false)]
+        [TestCase("InProgress", true, true, "New", false)]
+        [TestCase("InProgress", true, false, "Canceled", false)]
+        [TestCase("InProgress", true, false, "Completed", false)]
+        public async Task WHEN_order_is_SHOULD_return_Correct_CancelingStatus(string orderStatus,
```

Comment:
create separate file for this method and nice describe test cases
Cases for Order Status can be separated and have name
["Canceled", false]
["Completed", false]
["New", true]
When_Order_has_Status_SHOULD_return_correct_CanCancel(orderStatus, canCancel)